specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -2042,6 +1777,164 @@
         }
       }
     },
+    "HairProperties": {
+      "type": "object",
+      "description": "Properties describing hair attributes.",
+      "properties": {
+        "bald": {
+          "type": "number",
+          "description": "A number describing confidence level of whether the person is bald.",
+          "minimum": 0,
+          "maximum": 1
+        },
+        "invisible": {
+          "type": "boolean",
+          "description": "A boolean value describing whether the hair is visible in the image."
+        },
+        "hairColor": {
+          "type": "array",
+          "description": "An array of candidate colors and confidence level in the presence of each.",
+          "items": {
```

Comment:
>items [](start = 11, length = 5)

SAme comment here, it is better for stuff to have its own definitions, instead of defining them inline.

Reply:
Solved